docs/src/tricks.md

```diff
@@ -16,7 +16,7 @@ let
 in
 {
   stylix.image = pkgs.runCommand "dimmed-background.png" { } ''
-    ${pkgs.imagemagick}/bin/convert "${inputImage}" -brightness-contrast ${brightness},${contrast} -fill ${fillColor} $out
+    ${lib.getExe pkgs.imagemagick} "${inputImage}" -brightness-contrast ${brightness},${contrast} -fill ${fillColor} $out
```

Comment:
This has changed the program being used: should be `${lib.getExe' pkgs.imagemagick "convert"}`

Reply:
I believe `convert` is deprecated anyway; iirc upstream want users to switch to using `magik`?

Reply:
Yes although that is out of scope for this PR